pkg/crc/oc/oc.go

```diff
@@ -0,0 +1,30 @@
+package oc
+
+import (
+	"path/filepath"
+
+	"github.com/code-ready/crc/pkg/crc/constants"
+	crcos "github.com/code-ready/crc/pkg/os"
+)
+
+type OC struct {
+	OcBinaryPath   string
+	KubeconfigPath string
+}
+
+func GetOC(machineName string) OC {
+	oc := OC{
+		OcBinaryPath:   filepath.Join(constants.OcCacheDir, constants.OcBinaryName),
+		KubeconfigPath: filepath.Join(constants.MachineInstanceDir, machineName, "kubeconfig"),
+	}
+	return oc
+}
+
+func (oc OC) RunOcCommand(args ...string) (string, string, error) {
+	args = append(args, "--kubeconfig", oc.KubeconfigPath)
+	stdout, stderr, err := crcos.RunWithDefaultLocale(oc.OcBinaryPath, args...)
+	if err != nil {
+		return stdout, stderr, err
+	}
+	return stdout, stderr, nil
```

Comment:
This can be summed up as
```
args = append(args, "--kubeconfig", oc.KubeconfigPath)
return crcos.RunWithDefaultLocale(oc.OcBinaryPath, args...)
```

Reply:
if no further error processing is needed, return the function result directly.